Add Toggletip (#6154)
jpmorganchase/salt-ds: site/docs/components/toggletip/usage.mdx

---
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:
  exclude: true
data:
  $ref: ./#/data
---

## Using the component

### When to use

- To provide supplementary information for elements on a page that you can't add a Tooltip to, such as a section header.

### When not to use

- To provide supplementary information for an element that can have a tooltip e.g. [`Button`](/salt/components/button). In this case, use a [`Tooltip`](/salt/components/tooltip).
- To provide more complex information or user flows or present critical information that requires immediate action from the user. Instead use a [`Dialog`](/salt/components/dialog).

> - When the information is not supplementary and is needed to complete a task. Instead, add the information to the page so it's always visible.

Reviewer: ```suggestion
- When the information is needed for the user to complete a task. Instead, add the information to the page so it's always visible.
```
I feel like we say supplementary too much on this page
